### ISSUE TITLE
Error when spawning new scratchpad window

### ISSUE THREAD

**Issue by James-Randall-14** [hyprland-community/pyprland#24]
Trying to add some scratchpad windows to my hyprland config (built from source on Arch), encountering the following error:

```
Bug detected, please report on https://github.com/fdev31/pyprland/issues
Client address is invalid
Traceback (most recent call last):
  File "/usr/lib/python3.11/site-packages/pyprland/command.py", line 75, in _callHandler
    await getattr(plugin, full_name)(*params)
  File "/usr/lib/python3.11/site-packages/pyprland/plugins/scratchpads.py", line 468, in run_toggle
    await self.run_show(uid)
  File "/usr/lib/python3.11/site-packages/pyprland/plugins/scratchpads.py", line 519, in run_show
    await item.updateClientInfo()
  File "/usr/lib/python3.11/site-packages/pyprland/plugins/scratchpads.py", line 194, in updateClientInfo
    client_info = await get_client_props(addr="0x" + self.address)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/pyprland/plugins/scratchpads.py", line 52, in get_client_props
    assert len(addr) > 2, "Client address is invalid"
           ^^^^^^^^^^^^^
AssertionError: Client address is invalid
```
The contents of my pyprland.conf:
```
{
  "pyprland": {
    "plugins": [
      "scratchpads",
      "lost_windows"
    ]
  },
  "scratchpads": {
    "volume": {
      "command": "pavucontrol",
      "animation": "fromRight",
      "margin": 40,
      "size": "20% 40%",
      "position": "75% 55%",
      "lazy": true
    }
  }
}
```
And the relevant parts of my hyprland.conf:
````
# Scratchpad Configuration
# Add scratchpads in pyprland.conf

bind = $mainMod,V,exec,pypr toggle volume
windowrule = float,^(pavucontrol)$
windowrule = workspace special silent,^(pavucontrol)$
````

EDIT: Just noticed that I do not get this printout when I run `pypr toggle volume`, only when I launch the window with my Hyprland binding.

**Comment by fdev31** [comment]
Can you provide a bit more details ?
Is the log on start or when you try to show the scratch ?
Do you have an existing `pavucontrol` running which may prevent this one to fully start ?
Does it happen if you use another app which is quick to start, like a terminal ?

**Comment by James-Randall-14** [comment]
I built from source to install, without the AUR package. I will try the AUR package now. The issue occurs whenever I try to show a scatchpad, and I don't have an existing `pavucontrol` instance. And it does still occur when I launch `alacritty` instead of `pavucontrol`.
Thanks  : >

**Comment by fdev31** [comment]
Remove any "manual install" and try to play with the options "animation" & "lazy".
Then please the full logs (or at least some history before the problem) so I can have a look.

**Comment by fdev31** [comment]
I made a lot of changes to sanitize the code recently.
I have high hopes your issue comes from the JSON syntax (moving to TOML should fix it) or has been addressed in version 1.5.1.

**Comment by fdev31** [comment]
Please reopen with the requested information if you experience something similar with the latest version. I'm assuming this has been fixed.
